core/lib/makeswift/components/site-theme/sections/index.ts

```diff
@@ -0,0 +1,7 @@
+import { footer } from './footer';
+import { slideshow } from './slide-show';
+
+export default {
+  footer,
+  slideshow,
+};
```

Comment:
Do we need to introduce a `header` section here?

Reply:
I thought it was peculiar that we don't have header here as well, but I guess all of our header customization lives under Navigation and Banner: 
<img width="240" alt="theme props" src="https://github.com/user-attachments/assets/8c6380ee-9eca-4de5-9315-be14cd7d7e16" />

We could introduce a level of indirection here and group them under Sections/Header for consistency at the price of an extra click for the user. cc: @apledger

Reply:
Yeah, I guess it's a question of whether you're customizing the header section or the underlying navigation primitive. Same goes for the Card component and CardCarousel. I'd imagine we want both layers. Maybe I want a default for my Card but to make some tweaks to all cards in my CardCarousel?

Reply:
Right now you're customizing the banner and the navigation primitives, and there is no additional customization layer for the header in the code. That said, the site theme is specific to Catalyst and right now the only place where we use both primitives is the header, so personally I'd be okay with moving them from the Primitives section to Sections / Header for the sake of discoverability/at-first-glance consistency.

Reply:
We do have the header section in VIBES, independent of Catalyst. 

I say go with whichever you think it best 👍🏻

Reply:
Moved Banner and Navigation into the new Header component:

<img width="758" alt="Header" src="https://github.com/user-attachments/assets/2780880e-91c4-42a4-93a2-9755bf82d808" />